src/fish/data.ts

```diff
@@ -0,0 +1,5265 @@
+export interface GameDataFish {
+  /** The fish id */
+  id: number
+
+  /** The fish name */
+  name: string
+
+  /** The unlock items for the fish */
+  rarity: number
+
+  location: string
+
+  openWater: boolean
+
+  fishingHole: string[]
+
+  favoredBait: string
+
+  timeOfDay: string[]
+
+  fishingPower: number
+
+  achievement: {
+    achievementId: number
+    achievementBit: number
+  }
+
+  avidAchievement: {
+    achievementId: number
+    achievementBit: number
+  }
+}
+
+// prettier-ignore
+const GAME_DATA_FISHES: Array<GameDataFish> = [
+  // -- GENERATE-START --
+  {
+    name: 'Globefish',
+    id: 95894,
+    rarity: 1,
+    location: 'Seitung Province',
+    timeOfDay: ["Any"],
+    openWater: true,
+    fishingHole: ["None"],
+    favoredBait: 'Any',
+    fishingPower: 50,
+    achievement: {
+      achievementId: 6336,
+      achievementBit: 0
+    },
+    avidAchievement: {
+      achievementId: 6264,
+      achievementBit: 0
+    }
+  },
```

Comment:
To keep it consistent with the other data files:
- Output data for each fish in a single line, with no line breaks and using single spaces
- Output arrays as `['foo', 'bar']` instead of as `["foo","bar"]`